onecodex/analyses.py

```diff
@@ -1,5 +1,8 @@
+from __future__ import annotations
```

Comment:
Is this necessary for Python > 3.9?

Reply:
It's needed for postponed evaluation of annotations (e.g. deferred pandas imports)